crates/sncast/src/response/explorer_link.rs

```diff
@@ -0,0 +1,21 @@
+use super::print::OutputFormat;
+use crate::helpers::block_explorer::Service;
+
+pub trait OutputLink {
+    const TITLE: &'static str;
+
+    fn format_links(&self, service: Service) -> String;
```

Comment:
```suggestion
    fn format_links(&self, provider: Box<dyn LinkProvider>) -> String;
```

The you only use `.as_provider once`